hnn/visvolt.py

```diff
@@ -1,52 +1,40 @@
 import sys, os
+import numpy as np
+import pickle
+
 from PyQt5.QtWidgets import QMainWindow, QAction, qApp, QApplication, QToolTip, QPushButton, QFormLayout
 from PyQt5.QtWidgets import QMenu, QSizePolicy, QMessageBox, QWidget, QFileDialog, QComboBox, QTabWidget
 from PyQt5.QtWidgets import QVBoxLayout, QHBoxLayout, QGroupBox, QDialog, QGridLayout, QLineEdit, QLabel
 from PyQt5.QtWidgets import QCheckBox, QInputDialog
 from PyQt5.QtGui import QIcon, QFont, QPixmap
 from PyQt5.QtCore import QCoreApplication, QThread, pyqtSignal, QObject, pyqtSlot
 from PyQt5 import QtCore
-import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
 from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT as NavigationToolbar
 from matplotlib.figure import Figure
-import pylab as plt
 import matplotlib.gridspec as gridspec
-from neuron import h
-from run import net
-import paramrw
-import pickle
-from conf import dconf
-from gutils import getmplDPI
+from hnn_core import read_params
+
+from .qt_lib import getmplDPI
+from .paramrw import get_output_dir
+from .DataViewGUI import DataViewGUI
 
-if dconf['fontsize'] > 0: plt.rcParams['font.size'] = dconf['fontsize']
-else: dconf['fontsize'] = 10
+fontsize = plt.rcParams['font.size'] = 10
 
 # colors for the different cell types
 dclr = {'L2_pyramidal' : 'g',
         'L5_pyramidal' : 'r',
         'L2_basket' : 'w', 
         'L5_basket' : 'b'}
 
-ntrial = 1; tstop = -1; outparamf = voltpath = paramf = ''; 
-
 maxperty = 10 # how many cells of a type to draw
 
-for i in range(len(sys.argv)):
-  if sys.argv[i].endswith('.param'):
-    paramf = sys.argv[i]
-    tstop = paramrw.quickgetprm(paramf,'tstop',float)
-    ntrial = paramrw.quickgetprm(paramf,'N_trials',int)
-    outparamf = os.path.join(dconf['datdir'],paramf.split('.param')[0].split(os.path.sep)[-1],'param.txt')
-  elif sys.argv[i] == 'maxperty':
-    maxperty = int(sys.argv[i])
-
 if ntrial <= 1:
-  voltpath = os.path.join(dconf['datdir'],paramf.split('.param')[0].split(os.path.sep)[-1],'vsoma.pkl') 
+  voltpath = os.path.join(get_output_dir(), 'data', params['sim_prefix'], 'vsoma.pkl')
```

Comment:
in this file flake8 complains that many variables are undefined like `ntrial` ... might lead to errors.

On my PR, I get:

```py
(base) PHS008348:hnn mainak$ flake8 hnn/visvolt.py | grep undefined
hnn/visvolt.py:32:4: F821 undefined name 'ntrial'
hnn/visvolt.py:33:53: F821 undefined name 'params'
hnn/visvolt.py:35:53: F821 undefined name 'params'
hnn/visvolt.py:87:8: F821 undefined name 'tstop'
hnn/visvolt.py:87:36: F821 undefined name 'tstop'
hnn/visvolt.py:88:8: F821 undefined name 'i'
hnn/visvolt.py:97:10: F821 undefined name 'ntrial'
hnn/visvolt.py:103:62: F821 undefined name 'params'
hnn/visvolt.py:124:25: F821 undefined name 'paramf'
hnn/visvolt.py:135:44: F821 undefined name 'paramf'
hnn/visvolt.py:142:20: F821 undefined name 'ntrial'
```